common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -99,6 +99,7 @@
     private static final AtomicLong DIRECT_MEMORY_COUNTER;
     private static final long DIRECT_MEMORY_LIMIT;
     private static final ThreadLocalRandomProvider RANDOM_PROVIDER;
+    private static final int TRY_ALLOCATE_UNINITIALIZED_ARRAY;
```

Comment:
nit: maybe rename to make it more clear how it's used? Something like `UNINITIALIZED_ARRAY_ALLOCATION_THRESHOLD`?